signer/src/main.rs

```diff
@@ -261,14 +265,30 @@ async fn run_block_observer(ctx: impl Context) -> Result<(), Error> {
     block_observer.run().await
 }
 
-#[allow(dead_code)] // Remove when implemented
-async fn run_transaction_signer(_ctx: impl Context) -> Result<(), Error> {
-    // threshold: 2
-    // context_window: 10000
-    todo!()
+/// Run the transaction signer event-loop.
+#[tracing::instrument(skip(ctx))]
+async fn run_transaction_signer(ctx: impl Context) -> Result<(), Error> {
+    let config = ctx.config().clone();
+    let network = P2PNetwork::new(&ctx);
+
+    let signer = transaction_signer::TxSignerEventLoop {
+        network,
+        context: ctx.clone(),
+        context_window: 10000,
+        threshold: 2,
+        // TODO: Update when we have a blocklist impl
+        blocklist_checker: BlocklistClient::new(&ctx),
```

Comment:
This comment can be removed now.

Reply:
Sloppy :grimacing: 
Removed together with the merge commit [3958c8e](https://github.com/stacks-network/sbtc/pull/610/commits/3958c8ea34341e94a238136f0fc1dcd325325f92)